### PR TITLE
UseSqlLite

### PR DESCRIPTION
**Pull request by anasmilhem**
I Integrated SQL lite instead of using SQL server. It's much simpler and it can run on mac.
Download SqlLite from https://sqlitebrowser.org/ in order to open the db file once you run the program.

### CODE REVIEW

**Comment by zkhourdaji** [comment]
Very nice! I tested it and it works. I also tested deleting the database and running the project and it creates and seeds it as expected! Very good work!

File: Authentication2/Authentication2/appsettings.json [hunk 1]
```diff
@@ -5,7 +5,10 @@
     }
   },
   "ConnectionStrings": {
-    "DefaultConnection": "Server=localhost;Database=COSC481_Authentication;Trusted_Connection=True;MultipleActiveResultSets=true"
+
+    "DefaultWinConnection": "DataSource =C:\\Users\\Public\\DatabaseLite.db",
```
Comment (thonnissen) [inline Authentication2/Authentication2/appsettings.json]: is this the line of code that was the issue with Windows

Comment (thonnissen) [reply Authentication2/Authentication2/appsettings.json]: ok forget it I see it now

**Comment by zkhourdaji** [comment]
I will test this and merge to master

**Comment by zkhourdaji** [comment]
Very cool! Good job Anas and Sean. I tested it on windows and mac and it works perfectly. I will merge this to master

**Pull request merged**